src/ex1/scripts/domScripts.js

```diff
@@ -0,0 +1,195 @@
+const taskInput = document.querySelector(".task-input");
+const addTaskButton = document.querySelector(".task-add-btn");
+const tasksList = document.querySelector(".tasks-list");
+const filterOptiopn = document.querySelector(".filter-tasks");
+const searchInput = document.querySelector(".search-input");
+
+addTaskButton.addEventListener("click", onAddTaskClick);
+tasksList.addEventListener("click", onTaskButtonsClick);
+filterOptiopn.addEventListener("click", onFilterOptionChange);
+searchInput.addEventListener("keyup", onSearchInputKeyUp);
+
+taskInput.addEventListener("keypress", function (event) {
+  if (event.key === "Enter") {
+    event.preventDefault();
+    addTaskButton.click();
+  }
+});
+
+function onAddTaskClick(e) {
+  const isOk = canProceed();
+  if (isOk) {
+    const taskDiv = document.createElement("div");
+    taskDiv.classList.add("task");
+
+    const gripLines = createGridLines();
+    taskDiv.appendChild(gripLines);
+
+    const taskContent = createTaskContent();
+    taskDiv.appendChild(taskContent);
+
+    const completeTaskButton = createCompleteBtn();
+    taskDiv.appendChild(completeTaskButton);
+
+    const removeTaskButton = createRemoveBtn();
+    taskDiv.appendChild(removeTaskButton);
+
+    addTaskDivEventListeners(taskDiv);
+    tasksList.appendChild(taskDiv);
+    saveTaskToLocalStorage(taskContent.innerText, false);
+    toggleEmptyMsg();
+    taskInput.value = "";
+  }
+}
+
+function canProceed() {
+  const isAlertOn = isAlertShown();
+  if (taskInput.value === "") {
+    if (!isAlertOn) {
+      toggleAlert();
+    }
+    return false;
+  } else {
+    if (isAlertOn) {
+      toggleAlert();
+    }
+    return true;
+  }
+}
+
+function createGridLines() {
+  const gripLines = document.createElement("div");
+  gripLines.classList.add("grip-lines-icon");
+  gripLines.innerHTML = '<i class="fas fa-grip-lines"></i>';
```

Comment:
Don't create html elements by strings, just do `.createElement` if needed.
Also, why use the <i> element? it is not meant for such a case, it is more to distinguish certain text from other text in an 'italicized' format